js/interface.js

```diff
@@ -1193,6 +1193,38 @@ var DynamicLists = (function() {
       _this.handleTokensSelection();
       _this.loadTokenFields();
     },
+    sortDataSourcesListByCurrentAppId: function(dataSources) {
+      var groupedDataSources = [
+        {
+          id: 'currentAppDSs',
+          text: 'This app',
+          name: 'currentApp',
```

Comment:
`id` and `name` seem to serve very similar purpose here. Can we just keep one?

I prefer the value `currentApp` more than `currentAppDSs`.

Reply:
> id and name seem to serve very similar purpose here. Can we just keep one?

We can, but we will have to alter a little bit more the `customDSsearch` function so it worked correctly. Because now we show entries only if they have `id`, `text` and `name` keys in it.
Should we do that?

> I prefer the value currentApp more than currentAppDSs.
I will change it.